amazon-kinesis-client/src/test/java/software/amazon/kinesis/coordinator/SchedulerTest.java

```diff
@@ -317,17 +318,17 @@ public final void testInitializationFailureWithRetriesWithConfiguredMaxInitializ
     }
 
     @Test
-    public final void testMultiStreamInitialization() throws ProvisionedThroughputException, DependencyException {
+    public final void testMultiStreamInitialization() {
         retrievalConfig = new RetrievalConfig(kinesisClient, multiStreamTracker, applicationName)
                 .retrievalFactory(retrievalFactory);
         leaseManagementConfig = new LeaseManagementConfig(tableName, dynamoDBClient, kinesisClient,
                                                           workerIdentifier).leaseManagementFactory(new TestKinesisLeaseManagementFactory(true, true));
         scheduler = new Scheduler(checkpointConfig, coordinatorConfig, leaseManagementConfig, lifecycleConfig,
                 metricsConfig, processorConfig, retrievalConfig);
         scheduler.initialize();
-        shardDetectorMap.values().stream()
```

Comment:
For my own understanding. Why is .stream() omitted from a handful of the tests? I presume it's no (longer ?) necessary but would like to understand why.